core/ecschema-metadata/eslint.config.js

```diff
@@ -0,0 +1,32 @@
+const iTwinPlugin = require("@itwin/eslint-plugin");
+
+module.exports = [
+  {
+    files: ["**/*.ts"],
+    ...iTwinPlugin.configs.iTwinjsRecommendedConfig,
+  },
+  {
+    files: ["**/*.ts"],
+    rules: {
+      "radix": "warn",
+      "@typescript-eslint/explicit-member-accessibility": "warn",
+      "@itwin/no-internal-barrel-imports": [
+        "error",
+        {
+          "ignored-barrel-modules": [
+            "./src/ECObjects.ts"
+          ]
+        }
+      ]
+    }
+  },
+  {
+    files: [
```

Comment:
The old configuration from the package.json disables the no-internal-barrel-imports rule in the .tsx files as well. A quick change could be to make the file globs end in .ts*

Reply:
Yes, but there are no .tsx files in this monorepo I believe. So that's why we got rid of it.